spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,45 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          feed_uuid: feeds[0]
+        }
+      end
+    end
+
+    after(:all) do
+      require 'fileutils'
+      FileUtils.rm_rf "#{VCR.configuration.cassette_library_dir}/Inventory/tmp"
+    end
+
+    let(:cassette_name) do |e|
+      description = e.description
+      description
```

Comment:
You are right, it can be simplified to

```ruby
    let(:cassette_name) do |e|
      e.description
    end
```

There was an issue with rubocop. It was complaining that it can be simplified even more as

```ruby
let(:cassette_name, &:description)
```
=>

```bash
spec/integration/inventory_spec.rb:50:25: C: Pass &:description as an argument to let instead of a block.
    let(:cassette_name) do |e|
                        ^^^^^^

25 files inspected, 1 offense detected
```

But if I did that, it didn't work. So this is kind of a dummy workaround to rubocop not to complain. To be honest, I don't know why the `let(:cassette_name, &:description)` is not working, if you have any better workaround, I am happy to change that.